pkgs/development/python-modules/z3-solver/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "z3-solver";
+  version = "4.8.5.0";
+
+  src = fetchPypi {
```

Comment:
please include a link to the actual source in the correct repo/branch. It seems quite hard to find it.